htsget-config/README.md

```diff
@@ -347,6 +347,63 @@ ticket_server.cors.expose_headers = []
 Use `"Mirror"` to mirror CORS requests, and `"All"` to allow all methods, headers, or origins. The `ticket_server` table
 above can be replaced with `data_server` to configure CORS for the data server.
 
+### JWT Authorization
+
+One advantage of the htsget protocol is that it is possible to make decisions about which regions of files a user is allowed
+to access, as the protocol is able to return a subset of a genomic file in the URL tickets. Custom JWT authorization can be
+configured to enable this.
+
+Since htsget-rs is a primarily stateless service (except for caching), authorization is configured to call out to an arbitrary
+url to make decisions about a user. If this feature is configured, when a JWT is sent in the authorization header as a
+bearer token, htsget-rs finds a configurable key in the claims and sends a GET request to the URL contained in the key
+value to ask for authorization information about the user. The authorization server should then respond approving or
+denying the user access.
+
+The following options can be configured under the `auth` table to enable this:
+
+| Option                       | Description                                                                                                                                                                                                                                                                                                                          | Type             | Default                                                                                      |
+|------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------|----------------------------------------------------------------------------------------------|
+| `jwks_url`                   | The JSON web key sets url to fetch a `.well-known/jwks.json` file that validates the JWT token. This should not contain the `.well-known/jwks.json` component.                                                                                                                                                                       | URL              | Not set, either this option or `decode_public_key` must be set to validate JWTs.             | 
```

Comment:
I don't think this is correct - the `.well-known` conventions defines where the openid configuration is - which then itself can point to an arbitrary JWKS location. I'm not sure the convention is strong enough around the naming of the JWKS that we can rely on adding the suffix ourselves. 

e.g. google openid - `https://accounts.google.com/.well-known/openid-configuration` point to a JWKS of `https://www.googleapis.com/oauth2/v3/certs`

I think instead this URL should be the complete URL to the JWKS json.

Reply:
Fair enough, I thought maybe this convention applies for all, but I only really check AWS.